### PR TITLE
Remove transpose of rotations in QDA

### PR DESCRIPTION
**Pull request by youssef-emad**
Removing the unnecessary transpose enhances Shogun's performance to be same as scikit-learn.
check this [notebook](https://github.com/youssef-emad/shogun_notebooks/blob/master/QDA.Investigation.ipynb) for results before and after modification.

### CODE REVIEW

**Comment by karlnapf** [comment]
Hey are you aware that Shogun stores variables as column vectors, where numpy does row vectors?

**Comment by karlnapf** [comment]
Oh man this QDA code can be done much cleaner I think

**Comment by youssef-emad** [comment]
yeah I am aware of that but I don't understand your comment about numpy. and yeah of course the code can be cleaner but  I thought of this a quick solution of the accuracy issue.

**Comment by karlnapf** [comment]
my question is: is this really a bug or does it come from the fact that you did pass shogun features that were not in the right order.
If you pass something to a sklearn algortihm, you have to pass the same thing transposed to Shogun.
Meet me in irc if you have questions on this

**Comment by youssef-emad** [comment]
@karlnapf I made this mistake once but I corrected it after that. you can check this [notebook](https://github.com/youssef-emad/shogun_notebooks/blob/master/Classifiers.ipynb). I am passing the features the same way for KNN and Naive Bayes and the accuracy and decision boundaries are identical for SKlearn and Shogun.

**Comment by karlnapf** [comment]
So this is actually a bug?
Good catch then...

**Comment by youssef-emad** [comment]
Yes I believe it is.

**Comment by karlnapf** [comment]
Sweet! :)

**Comment by karlnapf** [comment]
Another thing to benchmark properly via the framework

**Comment by youssef-emad** [comment]
Thanks :smiley: 
I intend to continue adding benchmarks but I am waiting for the PR  to be merged to avoid conflicts in `config.yaml` file. I will add the tests needed tomorrow and ensure it get merged soon :) .
